rust/Cargo.toml

```diff
@@ -10,3 +10,6 @@ debug = false
 
```

Comment:
I added these patch directives.  It seems like the `bellman` patch is now not affecting the dependency graph!

Reply:
Fixed locally.